azure_sdk/azure_sdk.gemspec

```diff
@@ -19,6 +19,14 @@ Gem::Specification.new do |spec|
   spec.summary       = 'Official Ruby client library to consume Microsoft Azure services.'
   spec.homepage      = 'https://aka.ms/azure-sdk-for-ruby'
   spec.license       = 'MIT'
+  spec.metadata      = {
+    'bug_tracker_uri' => 'https://github.com/Azure/azure-sdk-for-ruby/issues',
+    'changelog_uri' => 'https://github.com/Azure/azure-sdk-for-ruby/blob/master/ChangeLog.md',
+    'documentation_uri' => 'https://azure.microsoft.com/en-us/develop/ruby/',
+    'homepage_uri' => 'https://aka.ms/azure-sdk-for-ruby',
+    'source_code_uri' => "https://github.com/Azure/azure-sdk-for-ruby/tree/azure-v#{Azure::VERSION}",
```

Comment:
I think you'd want this url to look like https://github.com/Azure/azure-sdk-for-ruby/tree/ARM-0.15.0 - notice "ARM" in the tag name.

Reply:
Thanks!